pkg/core/chain/chain.go

```diff
@@ -60,11 +60,10 @@ type Loader interface {
 
 // Ledger is the Chain interface used in tests.
 type Ledger interface {
-	CurrentHeight() uint64
-	ProcessSucceedingBlock(block.Block) error
+	ProcessSuccessiveBlock(block.Block) error
```

Comment:
Can we rename this into `TryNextConsecutiveBlockInSync`? `Successive` is quite an odd adjective